portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase

```diff
@@ -1107,6 +1107,577 @@ definition {
+                locator1 = "CreateObject#OBJECT_NAME");
+
+            Button.clickSave();
+
+            ObjectAdmin.goToFieldsTab();
+
+            Refresh();
+
+		 }
+
+		task("And Adding items to the created picklist"){
+            LexiconEntry.gotoAdd();
+
+            Type(
+                locator1 = "ObjectField#LABEL_INPUT",
+                value1 = "Filter");
+
+            Click(
+                key_sorting = "",
+                locator1 = "ObjectCustomViews#EDIT_SORTING");
+
+            Click(
+                key_fieldType = "Picklist",
+                locator1 = "ObjectField#TYPE_OPTION");
+
+            Click(locator1 = "ObjectCustomViews#SELECT_AUTO_COMPLETE");
+
+            Click(
+                key_optionName = "Transport",
+                locator1 = "ObjectAdmin#LAYOUT_ADD_FIELD_OPTION");
+
+            Button.clickSave();
+
+            ObjectCustomValidation.gotoTab(tabName = "Details");
+            Button.clickPublish();
+
+			}
+       
+        task ("When the user creates a new data set") {
+            Navigator.openURL();
+
+            DataSetAdmin.goToDataSetAdminPage();
+
+            DataSetAdmin.createDataSetEntryViaAPI(
+                dataSetName = "Dataset Test",
+                restApplication = "/data-set-manager/fields",
+                restEndpoint = "/",
+                restSchema = "FDSField");
+	
+        }
+
+        task ("And a data set view created via api") {
+            DataSetAdmin.createFDSViewViaAPI(
+                dataSetName = "Dataset Test",
+                dataSetViewDescription = "FDSViewDescription",
+                key_dataSetViewNameList = "View Fields Test");
+
+            DataSetAdmin.goToSpecificViewPage(
+                dataSetName = "Dataset Test",
+                key_viewName = "View Fields Test");
+        }
+
+        task ("When goes to Fields Tab.") {
+            DataSetAdmin.goToTab(tabName = "Filters");
+
+            LexiconEntry.gotoAdd();
+
+            Click(
+                key_fieldName = "Selection",
+                locator1 = "FormViewBuilder#OBJECT_FIELD_BUTTON_DROPDOWN");
+
+            Type(
+                key_text = "Name",
+                locator1 = "TextInput#ANY",
+                value1 = "Transport");
+
+            Click(
+                key_class = "btn-secondary",
+                key_index = 3,
+                locator1 = "Button#ANY_WITH_CLASS_AND_INDEX");
+
+            SelectFrameTop();
+
+            Click.javaScriptClick(
+                key_sortOption = "Relationship_c_fdsViewERC",
+                locator1 = "ObjectCustomViews#SORT_OPTION");
+
+            Select(
+                locator1 = "DataSet#SOURCE_OPTIONS_SELECT",
+                value1 = "Transport");
+        }
+
+        task ("And Select the option Multiple,Include radio button") {
+            Click.clickAtNotVisible(
+                key_radioOption = "Multiple",
+                locator1 = "CommerceEntry#RADIO_BUTTON");
```

Comment:
This should select the Single option and then select just one pre populated value from the picklist